src/main/java/greencity/dto/user_own_security/UserRegisterDto.java

```diff
@@ -0,0 +1,30 @@
+package greencity.dto.user_own_security;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.hibernate.validator.constraints.Length;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+
```

Comment:
@  Data already consists @RequiredArgsConstructor
You probaly meant @NoArgsConstructor  or @AllArgsConstructor?
https://projectlombok.org/features/Data